### DISCUSSION TITLE
How is `*.ddev.site` resolved without editing /etc/hosts?

### DISCUSSION THREAD

**Discussion by SuperFlySly** [ddev/ddev#5958] [Q&A]
How does ddev make it possible to create a domain such as projectname.ddev.site without modifying /etc/hosts? Does docker offer a special function for this which is perhaps available via the docker API?

**Comment by rfay** [comment]
No, it's a DNS wildcard. So `*.ddev.site` always evaluates to 127.0.0.1. Most important, it's not something that docker could impact, because it's your *browser* that has to look up this hostname.
